src/constants/routes.js

```diff
@@ -0,0 +1,2 @@
+export const channelRoute = '/chat/channel/:channelId([0-9]+):separator(:)?:messageId?/:slug?'
```

Comment:
if file is already called "routes", why do you duplicate Route in constant names?